### DISCUSSION TITLE
Is there a way to get a notification when a resource is suspended?

### DISCUSSION THREAD

**Discussion by marc-barry** [fluxcd/flux2#4072] [Q&A]
We use the Notification Controller to send us alerts to Slack when certain issues happen within our clusters running Flux. I'm looking for a way to get an alert when someone suspends Flux. For example, assume the following is run:

```
flux suspend kustomization flux-system
► suspending kustomization flux-system in flux-system namespace
✔ kustomization suspended
```
I'd like a notification to come to Slack letting us know that someone suspended a Flux resource. I think https://github.com/fluxcd/flux2/discussions/3872 is a great idea as well and that metadata could be propagated along with the alert.

**Comment by havard024** [comment]
I've asked a similar question a while back, not sure if this is what you were after, but it is possible to configure alerts on suspend / resume: https://github.com/fluxcd/flux2/discussions/3144#discussioncomment-3732610
